opentelemetry-auto-instrumentation/Gemfile

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+source 'https://rubygems.org'
+
+# avoid use gemspec to avoid circular load
```

Comment:
I am confused about this. Aren't these already declared in the gemspec?

Reply:
Simplified the Gemfile